pkgs/data/misc/nixos-artwork/wallpapers.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchurl }:
 
 let
-  mkNixBackground = { name, src, description }:
+  mkNixBackground = { name, src, description, installPhase ? "" }:
```

Comment:
Not 100% confident in the cleanliness of this approach, but does work. Any alternatives?

Reply:
I guess it is ok. Alterantives would be `overrideAttrs` or adding `runHook postInstall` in installPhase and inheriting `postInstall`.